### PR TITLE
Use exit code 3 for errors, 1 for failures

### PR DESCRIPTION
**Pull request by andrewbranch**

### CODE REVIEW

File: packages/history/scripts/generateFull.ts [hunk 1]
```diff
@@ -29,8 +29,6 @@ const excludedSpecs = [
   "next@13.4.4", // File not found: /node_modules/next/types/misc.d.ts
   "next@13.4.7", // File not found: /node_modules/next/types/misc.d.ts
   "next@13.4.12", // File not found: /node_modules/next/types/misc.d.ts
-  "moment@2.29.1", // Invalid gzip data
-  "moment@2.29.3", // Invalid gzip data
```
Comment (andrewbranch) [inline packages/history/scripts/generateFull.ts:33]: These were fixed by #103

**Comment by jakebailey** [comment]
Hello drive by reviewer here; I would suggest choosing something < 125; 126 and 127 are special codes in the shells (which I learned from https://git-scm.com/docs/git-bisect#_bisect_run)

**Comment by jakebailey** [comment]
on second thought, it's 125 that's special, 126/127 are just errors to bisect, so, ignore me have a nice day

**Comment by andrewbranch** [comment]
I chose 127 because I feel like I’ve definitely seen it for errors, but the reason I’ve seen it is probably that it _is_ special to shells https://tldp.org/LDP/abs/html/exitcodes.html. I’ll pick something else 😄

**Pull request merged**